### PR TITLE
Added single node HC deployment guide

### PR DESCRIPTION
**Pull request by sabose**
Signed-off-by: Sahina Bose <sabose@redhat.com>

Fixes issue # (delete if not relevant)

Changes proposed in this pull request:

-

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

**Comment by sabose** [comment]
@doron-fediuck @sandrobonazzola @yanivlavi @akashihi - Please review and provide your inputs

File: source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+## Pre-requisites
+
+* A single host with  Enterprise Linux 7 or oVirt Node. Refer [Enterprise Linux Hosts](../install-guide/chap-Enterprise_Linux_Hosts) or [oVirt Nodes](../install-guide/chap-oVirt_Nodes)
+
+* You must have at least 2 interfaces on the host, so that the frontend and backend traffic can be separated out. Having only one network will cause the engine monitoring, client traffic, gluster I/O traffic to all run together and interfere each other. To segregate the backend network, the gluster cluster is formed using the backend network addresses, and the nodes are added to the engine using the frontend network address.
+
+* You must have a fully qualified domain name prepared for your Engine and the host. Forward and reverse lookup records must both be set in the DNS. **The Engine should use the same subnet as the management network.**
+
+* You must have configured passwordless ssh between the host to itself. gdeploy uses Ansible playbooks and the ability to remotely execute commands is a pre-requisite.
```
Comment (mykaul) [inline source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md]: how?

File: source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+        # yum install http://resources.ovirt.org/pub/yum-repo/ovirt-release42.rpm
+
+2. Install gdeploy and cockpit-ovirt that will provide a UI for the installation of Hosted Engine. gdeploy is a wrapper tool around Ansible that helps to setup gluster volumes. vdsm-gluster is used to manage gluster from oVirt, and pulls in all the required gluster dependencies.
+     
+        # yum install gdeploy cockpit-ovirt-dashboard vdsm-gluster
+
+3. Install the oVirt Engine Virtual Appliance package for the Engine virtual machine installation:
+
+        # yum install ovirt-engine-appliance
```
Comment (mykaul) [inline source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md]: Unify this with the previous 'yum install' command?

File: source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+## Deploying on oVirt Node based Hosts
+
+**oVirt Node contains all the required packages to set up the hyperconverged environment.**
+Refer [oVirt Nodes](../install-guide/chap-oVirt_Nodes) for instructions on installing oVirt Node on your hosts. You can proceed to setting up the hyperconverged environment if you have 3 oVirt Node based hosts.
```
Comment (mykaul) [inline source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md]: 1. Refer -> Refer to
2. Why do we need 3 nodes here?

File: source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+
+### Setting up the hyperconverged environment
+
+#### Installing and setting up gluster volume
+
+Gluster volumes needs to be created first prior to the Hosted Engine installation flow. The volume that's created is used to host the Hosted Engine VM. Use the below gdeploy configuration file as a template to create volumes and configure the host.
```
Comment (mykaul) [inline source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md]: needs -> need

File: source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+transport=tcp
+key=storage.owner-uid,storage.owner-gid,features.shard,performance.low-prio-threads,performance.strict-o-direct,network.remote-dio,network.ping-timeout,user.cifs,nfs.disable,performance.quick-read,performance.read-ahead,performance.io-cache,cluster.eager-lock
+value=36,36,on,32,on,off,30,off,on,off,off,off,enable
+brick_dirs=/rhgs/brick3/data
+ignore_volume_errors=no
+```
+
+Once the configuration file is edited (the @ @ replaced), the gluster environment can be setup using
+```
+gdeploy -c gdeploy.conf
```
Comment (mykaul) [inline source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md]: Why are we not promoting the Cockpit-based installation?

File: source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md (hunk 1 above)
Comment (sabose) [reply source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md]: I can add the steps to configure this and refer here

File: source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md (hunk 2 above)
Comment (sabose) [reply source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md]: Done

File: source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md (hunk 3 above)
Comment (sabose) [reply source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md]: Thanks - copy & paste error.

File: source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md (hunk 4 above)
Comment (sabose) [reply source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md]: Done

File: source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md (hunk 5 above)
Comment (sabose) [reply source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md]: For single node - we donot have the Cockpit based UI.

**Pull request merged**